ui/app/scripts/shared/services.js

```diff
@@ -59,7 +59,7 @@ define(['angular', 'xregexp'], function (angular) {
       })
       .factory('xdVersionInfo', function ($resource, $rootScope, XDUtils) {
         console.log('xdVersionInfo');
-        var xdVersionInfoPromise =  $resource($rootScope.xdAdminServerUrl + '/meta/version', {}, {
+        var xdVersionInfoPromise =  $resource($rootScope.xdAdminServerUrl + '/management/info', {}, {
```

Comment:
At some point of time we need to get rid of `xd` in this code base. May be sooner.